packages/cashier/src/pages/payment-agent/payment-agent-deposit-details/payment-agent-deposit-details.tsx

```diff
@@ -21,33 +21,22 @@ const PaymentAgentDepositDetails = ({ payment_agent }: TPaymentAgentDepositDetai
         );
     };
 
-    const PaymentAgentEmailDetails = () => {
+    const PaymentAgentTransferLimitDetails = () => {
         return (
-            <PaymentAgentDetail
-                action='mailto'
-                icon='IcEmailOutlineNew'
-                rel='noopener noreferrer'
-                target='_blank'
-                has_red_color
-                title={localize('Email')}
-            >
-                {payment_agent.email}
-            </PaymentAgentDetail>
-        );
-    };
-
-    const PaymentAgentMinimumWithdrawalDetails = () => {
-        return (
-            <PaymentAgentDetail icon='IcCashierMinimumWithdrawal' title={localize('Minimum withdrawal')}>
-                <Money amount={payment_agent.min_withdrawal || ''} currency={payment_agent.currency} show_currency />
-            </PaymentAgentDetail>
-        );
-    };
-
-    const PaymentAgentMaximumWithdrawalDetails = () => {
-        return (
-            <PaymentAgentDetail icon='IcCashierMaximumWithdrawal' title={localize('Maximum withdrawal')}>
-                <Money amount={payment_agent.max_withdrawal || ''} currency={payment_agent.currency} show_currency />
+            <PaymentAgentDetail icon='IcAccountTransfer' title={localize('Transfer limit')}>
+                <>
+                    <Money
+                        amount={payment_agent.min_withdrawal || ''}
+                        currency={payment_agent.currency}
+                        show_currency
+                    />{' '}
```

Comment:
wrap in a span and give margin left/right probably

Reply:
Sure will consider this, thanks

Reply:
Yes, adding span is better practice.

Reply:
Updated